### ISSUE TITLE
Zoom using an additional button

### ISSUE THREAD

**Issue by nizhnikovskiy** [amcharts/amcharts5#45]
Is it possible to somehow implement zoom of the map using Ctrl/Cmd button, how is it implemented in google maps? 
https://developers.google.com/maps/documentation/javascript/overview#The_Hello_World_of_Google_Maps_v3

When you try to scroll, the message "Use ctrl + scroll to zoom the map" is displayed (or trigger some event), but it doesn't prevent the rest of page from scrolling. Using Ctrl/Cmd + MouseWheel the map scaling works correctly, the rest of the page stops scrolling

**Comment by samkristoff** [comment]
I'd use this feature.

**Comment by samkristoff** [comment]
@nizhnikovskiy Thank you for posting that example.  I was able to adapt it to work on my XY chart.  I didn't notice any bugs related to scrolling while hovering over a tooltip on the XY chart.  

In my case I'm using a 'busy' overlay div to gray out the chart (very similar to the no-scroll div in your example).  When the user ctrl+scroll while the chart is busy (ie the busy div is visible) it zooms the browser.  I tried to catch and prevent this but my first couple attempts didn't work so I just switched to shift+scroll to zoom the chart.

Thanks again for sharing your example.

**Comment by martynasma** [comment]
You guys might find this useful:
https://www.amcharts.com/docs/v5/charts/xy-chart/zoom-and-pan/#Control_scroll
https://www.amcharts.com/docs/v5/charts/map-chart/map-pan-zoom/#Control_scroll

**Comment by martynasma** [comment]
Hey guys, quick update. I updated the above sections to include option to show Google-like overlay with a message informing user they need to use CTRL to zoom:

![image](https://user-images.githubusercontent.com/1458520/142577773-7baa07ec-a68f-46b4-878f-ea6559d86895.png)

Hope you find it useful.

**Comment by nizhnikovskiy** [comment]
In your example this works https://codepen.io/team/amcharts/pen/vYJPYbZ not quite right. It works by 
tracking a Ctrl/Meta button, but this key can be pressed when the browser is out of focus, so JS won't catch this event. As an option, we can check if a Ctrl/Meta button is pressed in a wheel event, but then it's impossible to set a wheel property of the map correctly and enable/disable zoom of the map.

**Comment by martynasma** [comment]
Yeah, agreed. It will not work reliably in iframe (unless you click on it), but should be good to go on regular pages.

We might consider adding a setting which would disable page scroll only if ctrl key is down, even if element is "wheelable".

**Comment by martynasma** [comment]
Indeed. Well, then then you will probably need to explicitly set the colors:

```
dropFromOpenState: { properties: { fill: "#67b7dc", stroke: "#67b7dc" } },
riseFromOpenState: { properties: { fill: "#67b7dc", stroke: "#67b7dc" } },
```

**Comment by nizhnikovskiy** [comment]
Ok, thanks. As an option, you can allow/disallow wheel of map according to condition specified in a some callback. This will add more flexibility

**Comment by martynasma** [comment]
That's actually a pretty good idea. We'll consider it. Thanks.

**Comment by martynasma** [comment]
We've just released 5.0.13 which does not cancel wheel event, so we more options to set up our own wheel logic.

The following docs have been updated to reflect that:
https://www.amcharts.com/docs/v5/charts/xy-chart/zoom-and-pan/#Control_scroll
https://www.amcharts.com/docs/v5/charts/map-chart/map-pan-zoom/#Control_scroll
